docs/mcmc/index.rst

```diff
@@ -298,34 +318,58 @@ the results match reality::
     >>> cdf.prepare(pvals[2, :] - truth.ypos.val, r'$\Delta y$')
     >>> cdf.plot(clearwindow=False)
     >>> plt.title('')
-
-.. image:: ../_static/mcmc/mcmc_cdf_xpos.png
+    >>> plt.subplots_adjust(hspace=0.4)
 
 Scatter plot
 ------------
 
-::
+.. plot::
+    :include-source:
+    :context: close-figs
 
     >>> plt.scatter(pvals[0, :] - truth.r0.val,
     ...             pvals[4, :] - truth.alpha.val, alpha=0.3)
     >>> plt.xlabel(r'$\Delta r_0$', size=18)
-    >>> plt.ylabel(r'$\Delta \alpha$', size=18)
-
-.. image:: ../_static/mcmc/mcmc_scatter_r0_alpha.png
+    >>> _ = plt.ylabel(r'$\Delta \alpha$', size=18)
    
 This can be compared to the
-:py:class:`~sherpa.plot.RegionProjection` calculation::
+:py:class:`~sherpa.plot.RegionProjection` calculation:
 
-    >>> plt.scatter(pvals[0, :], pvals[4, :], alpha=0.3)
+.. plot::
+    :include-source:
+    :context: close-figs
 
+    >>> plt.scatter(pvals[0, :], pvals[4, :], alpha=0.3)
     >>> from sherpa.plot import RegionProjection
     >>> rproj = RegionProjection()
     >>> rproj.prepare(min=[95, 1.8], max=[150, 2.6], nloop=[21, 21])
     >>> rproj.calc(f, mdl.r0, mdl.alpha)
     >>> rproj.contour(overplot=True)
-    >>> plt.xlabel(r'$r_0$'); plt.ylabel(r'$\alpha$')
+    >>> _ = plt.xlabel(r'$r_0$'); plt.ylabel(r'$\alpha$')
+
+Connections to ArviZ
+--------------------
+`ArviZ <https://python.arviz.org>`_ is a Python package for exploratory analysis of Bayesian models.
+It serves as a backend-agnostic tool for diagnosing and visualizing Bayesian inference and
+provides different plotting and statistical diagnostic functions for MCMC chains, that
+are not implemented in Sherpa itself. If the ArviZ package is installed,
+`~sherpa.sim.mcmc_to_arviz` can be used to convert the MCMC results to an ArviZ InferenceData object.
```

Comment:
```suggestion
`~sherpa.sim.mcmc_to_arviz` can be used to convert the MCMC results to an `ArviZ InferenceData object <https://python.arviz.org/en/latest/api/generated/arviz.InferenceData.html>`_.
```

[although this does tie us to the "current" sphinx rules for how their pages are generated for a particular term, I don't see that changing for a while]

assuming I've not messed up the markup

Reply:
Even better - I'm loading intersphinx for arvix now for exactly that reason. I just forgot to mark this up!